metrics.json

```diff
@@ -0,0 +1,25 @@
+{
+    "metrics": {
```

Comment:
Is this `metrics` entry needed? It only has sense if we add something else than `metrics` maybe